src/main/java/com/cinema/mapper/GenericMapper.java

```diff
@@ -0,0 +1,9 @@
+package com.cinema.mapper;
+
+public interface GenericMapper<T, K, V> {
+    T mapFromReqDtoToEntity(K dto);
+
+    K mapToReqDto(T entity);
```

Comment:
Why do you need map entity to ReqDto? You receive it from frontend and ReqDto have to be mapped to entity, not vice verse